Simplified/NYPLReaderReadiumView.m

```diff
@@ -343,6 +354,12 @@ - (void) openPageRight {
   [self sequentiallyEvaluateJavaScript:@"ReadiumSDK.reader.openPageRight()"];
 }
 
+/// Toggles user interaction to ensure text selections are cleared.
+- (void)clearTextSelection {
+  self.webView.userInteractionEnabled = !self.webView.userInteractionEnabled;
+  self.webView.userInteractionEnabled = !self.webView.userInteractionEnabled;
+}
```

Comment:
If you were repeating code in more than two places, I would refactor that into a method next time.